.circleci/config.yml

```diff
@@ -7,14 +13,35 @@ jobs:
       - checkout
       - run: make test 
   build:
-    machine:
-      docker_layer_caching: true
+    executor: docker-publisher
     steps:
       - checkout
-      - run: make build 
+      - run: make build
+      - run:
+          name: publish docker image with branch
+          command: |
+            docker tag contentful-labs/coredns-nodecache:latest $IMAGE_NAME:$CIRCLE_BRANCH
```

Comment:
we only want to tag latest if we are in the master branch?

Reply:
yes that is how this works, see https://github.com/contentful-labs/coredns-nodecache/pull/11/files#diff-1d37e48f9ceff6d8030570cd36286a61R46-R47